src/Watchers/DuplicateQueryWatcher.php

```diff
@@ -43,6 +43,17 @@ public function register(): void
         });
     }
 
+    private function cleanupBindings(array $bindings): array
+    {
+        return array_map(function ($binding) {
+            if ($binding instanceof \DateTime) {
```

Comment:
Maybe check for the `DateTimeInterface` instead in case people use immutable dates, currently this case would not be caught by this.

Reply:
@abenerd you're right! I'm pushing a commit to fix this